aten/src/ATen/native/TensorShape.cpp

```diff
@@ -840,6 +840,22 @@ Tensor repeat(const Tensor& self, IntArrayRef repeats) {
   return result;
 }
 
+Tensor tile(const Tensor& self, IntArrayRef reps){
+  // If self.size() > len(reps), reps is promoted to self.size() by pre-pending
+  // 1’s to it to keep the same behaviour as `numpy.tile`.
+  // Thus for a tensor of shape (2, 3, 4, 5), a dims of (2, 2) is treated
+  // as (1, 1, 2, 2).
+  int64_t size_diff = self.dim() - (int64_t)reps.size();
+  if (size_diff > 0){
+    std::vector<int64_t> new_reps(size_diff, 1);
+    for(int i = 0; i < reps.size(); ++i)
```

Comment:
Egyptian-style braces for all if statements and for loops, even if their bodies are a single line